readthedocs/api/v3/serializers.py

```diff
@@ -550,3 +560,46 @@ def get_from_url(self, obj):
     def get_to_url(self, obj):
         # Overridden only to return ``None`` when the description is ``''``
         return obj.to_url or None
+
+
+class EnvironmentVariableLinksSerializer(BaseLinksSerializer):
+    _self = serializers.SerializerMethodField()
+    project = serializers.SerializerMethodField()
+
+    def get__self(self, obj):
```

Comment:
2 underscsores is weird here.

Reply:
Agree. This is because we are using `_self` as our attribute since it's kind of a "special name". Then, we need `get_` and the attribute...